src/cli/go.js

```diff
@@ -11,15 +11,21 @@ function die(std, message) {
   return 1;
 }
 
+const noLoadArg = "--no-load";
+
 const goCommand: Command = async (args, std) => {
-  if (args.length !== 0) {
-    return die(std, "usage: sourcecred go");
-  }
   const commandSequence = [
     {name: "load", command: load},
     {name: "graph", command: graph},
     {name: "score", command: score},
   ];
+
+  if (args.length === 1 && args[0] === noLoadArg) {
+    commandSequence.shift();
```

Comment:
IMO this is a little bit cute and would generally prefer something more explicit, but this command is so trivial that I think its fine.